lib/presentation/journey/todo/todo_list/todo_list_screen.dart

```diff
@@ -11,31 +16,89 @@ class TodoListScreen extends StatefulWidget {
 }
 
 class _TodoListScreenState extends State<TodoListScreen> {
+  TodoBloc _todoBloc;
   @override
   void initState() {
     super.initState();
+    _todoBloc = Injector.resolve<TodoBloc>()..add(TodoFetchEvent());
   }
 
   @override
   void dispose() {
     super.dispose();
+    _todoBloc?.close();
```

Comment:
The same todo bloc is accessed by create todo screen. 
So if we are closing here, then create todo screen will throw error while accessing todoBloc